tests/EmitTest.elm

```diff
@@ -29,30 +61,12 @@ javascript =
                             |> typed
                             |> JS.emitExpr
                             |> Expect.equal output
-
-            typed : Typed.Expr_ -> Typed.Expr
-            typed expr =
-                ( expr, Type.Int )
-
-            typedInt : Int -> Typed.Expr
-            typedInt int =
-                typed (Literal (Int int))
-
-            typedString : String -> Typed.Expr
-            typedString str =
-                typed (Literal (String str))
-
-            typedBool : Bool -> Typed.Expr
-            typedBool bool =
-                typed (Literal (Bool bool))
           in
-          describe "emitExpr"
+          describe "emitExpr_"
             [ describe "Int"
                 (List.map runTest
                     [ ( "positive int", Literal (Int 42), "42" )
                     , ( "negative int", Literal (Int -998), "-998" )
-                    , -- Elm wat
-                      ( "negative zero int", Literal (Int (negate 0)), "0" )
```

Comment:
I'd like this test to come back :grimacing: similarly with the comment in the float test